### PR TITLE
Feature/173 Modified the theme of the app

### LINKED ISSUE #173: Modify the theme of the app + the company name

### PR DESCRIPTION
**Pull request by jmezaber**
### Example of what some pages look like now:

Login page
Screen Shot 2021-04-14 at 2 48 50 PM

Inventory page
Screen Shot 2021-04-14 at 2 55 40 PM

Financials
Screen Shot 2021-04-14 at 2 58 26 PM

closes #173 #133

### CODE REVIEW

File: bike-erp/src/components/Triggers/TriggersStyles.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { makeStyles } from "@material-ui/core";
 
 const useStyles = makeStyles((theme) => ({
     trigger: {
-      background: "#DDDDDD",
+      background: "rgb(240, 240, 240)",
```
Comment (bowsong) [inline bike-erp/src/components/Triggers/TriggersStyles.ts:5]: Can you use Hexadecimals for consistency

**Review by daniela-venuta — APPROVED** [review]

File: bike-erp/src/index.css [hunk 2]
```diff
@@ -5,7 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  background-color: rgb(221, 221, 221);
+  background-color: rgb(240, 240, 240);
```
Comment (bowsong) [inline bike-erp/src/index.css:8]: Here too

File: bike-erp/src/components/Triggers/TriggersStyles.ts (hunk 1 above)
Comment (bowsong) [inline bike-erp/src/components/Triggers/TriggersStyles.ts:5]: Can you use hexadecimals for consistency

File: bike-erp/src/pages/Inventory/InventoryStyles.ts [hunk 3]
```diff
@@ -8,12 +8,12 @@ const useStyles = makeStyles((theme) => ({
     fontSize: "200%",
   },
   background: {
-    background: "#DDDDDD",
+    background: "rgb(240, 240, 240)",
```
Comment (bowsong) [inline bike-erp/src/pages/Inventory/InventoryStyles.ts:11]: Here too please

**Review by bowsong — CHANGES_REQUESTED** [review]

File: bike-erp/src/index.css (hunk 2 above)
Comment (jmezaber) [reply bike-erp/src/index.css]: done!

File: bike-erp/src/components/Triggers/TriggersStyles.ts (hunk 1 above)
Comment (jmezaber) [reply bike-erp/src/components/Triggers/TriggersStyles.ts]: done!

File: bike-erp/src/pages/Inventory/InventoryStyles.ts (hunk 3 above)
Comment (jmezaber) [reply bike-erp/src/pages/Inventory/InventoryStyles.ts]: done!

**Review by bowsong — APPROVED** [review]

**Pull request merged**